corehq/apps/reminders/templates/reminders/partial/scheduled_reminders.html

```diff
@@ -83,13 +165,7 @@
     </style>
 {% endblock %}
 
-{% block subsection-title %}
-<li class="active">
-    <a href="#">Reminder Calendar</a>
-</li>
-{% endblock %}
-
-{% block main_column %}
+{% block page_content %}
     {% include "style/bootstrap2/partials/time_notice.html" %}
```

Comment:
I'm not sure what should be done with this time notice. It seems to me that time notice is compatible with b3 but definitely bootstrap2 in path can be confusing.